inspec/a2-iam-v2-upgrade-legacy-v1-policies/controls/base.rb

```diff
@@ -0,0 +1,322 @@
+# encoding: utf-8
+# copyright: 2018, Chef Software, Inc.
+# license: All rights reserved
+
+DEFAULT_POLICY_IDS = [
+    "administrator-access",
+    "editor-access",
+    "viewer-access",
+    "ingest-access"
+]
+
+DEFAULT_ROLE_IDS = [
+    "owner",
+    "editor",
+    "viewer",
+    "ingest",
+    "project-owner"
+]
+
+include_controls 'a2-api-integration' do
+  # stuff that changed with the changes in default profiles
+  # see authz-access-control-1 for the IAM v2 behaviour
+  skip_control 'authz-access-control-iam-v1'
+
+  # this is using the v1 policy API, which will be disabled when using v2
+  skip_control 'authz-api-crud-1'
+end
+
+# these must be run after a2-api-integration because those
+# tests assume that the legacy policies exist
+include_controls 'remove-legacy-policies'
+
+title 'IAM v2.1 migration from v1 API integration tests'
+
+control 'upgrade-iam-v2-1' do
+  title 'IAM v2 Policies'
+  desc 'test IAM v2 policy API post migration'
+
+  describe 'list policies' do
+    it 'includes the legacy policies we expect' do
+      resp = automate_api_request('/apis/iam/v2beta/policies')
+      expect(resp.http_status).to eq 200
+
+      all_policies = resp.parsed_response_body[:policies]
+      legacy_policies = all_policies.select{ |p| /^\[Legacy\]/.match(p[:name]) }
+      expect(legacy_policies.length).to eq 9
+    end
+  end
+end
+
+control 'upgrade-iam-v2-2' do
+  title 'authz_access_control variants for IAM v2'
+  desc <<EOF
+In a2-api-integration, we've singled out a few checks whose results differ for
+IAM v2. In this control, they're included with their NEW results.
```

Comment:
Could you help me understand what this message is trying to convey? Perhaps I am missing some context, but I don't follow.

Reply:
I didn't write this (at least not since the several months ago it was written) xD I think it's trying to say that these tests are different from v1.